.github/CODEOWNERS

```diff
@@ -341,7 +341,7 @@
 # ServiceOwners:                                     @kushagraThapar @TheovanKraay @pjohari-ms
 
 # PRLabel: %Cosmos
-/sdk/cosmos/                                         @kushagraThapar @FabianMeiswinkel @kirankumarkolli @xinlian12 @milismsft @aayush3011 @simorenoh @jeet1995 @Pilchie
+/sdk/cosmos/                                         @kushagraThapar @FabianMeiswinkel @kirankumarkolli @xinlian12 @milismsft @aayush3011 @simorenoh @jeet1995 @Pilchie @nehrao1
```

Comment:
It might be worth creating a GitHub Team and making it a codeowner similar to the one we have for other libraries. 

See [Monitor Team](https://github.com/orgs/Azure/teams/azure-sdk-write-monitor-data-plane) for example. Then, in codeowners file, you can just [reference this team](https://github.com/Azure/azure-sdk-for-java/blob/main/.github/CODEOWNERS#L568).

Reply:
@srnagar interesting, didn't know this. I will look into it. 
thanks Srikanta!